.travis/update_formula.sh

```diff
@@ -18,24 +12,32 @@ setup_git() {
     git config --global user.name "Raiden Network"
 }
 
+clone_repo() {
+    git clone git@github.com:raiden-network/homebrew-raiden
+}
+
 update_formula() {
-    UPDATED_SHA256=$(openssl sha -sha256 dist/archive/raiden-${TRAVIS_TAG}-macOS.zip)
+    UPDATED_SHA256=$(openssl dgst -sha256 dist/archive/raiden-${TRAVIS_TAG}-macOS.zip)
     FORMULA_FILE="homebrew-raiden/raiden.rb"
 
-    sed -i .bak "s/[0-9]\.[0-9]\.[0-9]/${TRAVIS_TAG/v/}/g" $FORMULA_FILE
-    sed -i .bak "s/sha256 \"[a-f0-9]\{64\}\"/sha256 \"${UPDATED_SHA256: -64}\"/g" $FORMULA_FILE
+    sed -i .bak -E "s/[0-9]+\.[0-9]+\.[0-9]+/${TRAVIS_TAG/v/}/g" $FORMULA_FILE
+    sed -i .bak -E "s/sha256 \"[a-f0-9]\{64\}\"/sha256 \"${UPDATED_SHA256: -64}\"/g" $FORMULA_FILE
 
     rm $FORMULA_FILE.bak
 }
 
 commit_formula_file() {
+    pushd homebrew-raiden
```

Comment:
I have not seen `pushd/popd/dirs` in such a long time!

Reply:
Seemed like a good idea considering we had so many errors with scripts assuming the wrong working dir on travis ;)